template/template.go

```diff
@@ -34,6 +34,19 @@ var (
 	ErrMissingReaderFunction = errors.New("template: missing a reader function")
 )
 
+var (
+	sprigFuncMap template.FuncMap
+)
+
+func init() {
+	sprigFuncMap = make(template.FuncMap, len(sprig.TxtFuncMap()))
+	// Add the Sprig functions to the funcmap
+	for k, v := range sprig.TxtFuncMap() {
```

Comment:
Using `HemerticTxtFuncMap` instead of `TxtFuncMap` would remove some sprig functions that may be problematic.

Reply:
Done!